vignettes/datatable-intro.Rmd

```diff
@@ -250,6 +250,50 @@ The function `length()` requires an input argument. We just need to compute the
 
 This type of operation occurs quite frequently, especially while grouping (as we will see in the next section), to the point where `data.table` provides a *special symbol* `.N` for it.
 
+### g) Handle non-existing elements in `i`
+
+#### -- What happens when querying for non-existing elements?
+
+When querying a `data.table` for elements that do not exist, the behavior differs based on the method used.
+
+```r
+dt <- data.table(x = letters[1:3], y = LETTERS[1:3])
```

Comment:
The vignette uses the flight dataset so it would be ideal to have an example using that instead of creating something seperate at this point